pyia/data.py

```diff
@@ -427,8 +432,18 @@ def get_skycoord(self, distance=None, radial_velocity=None):
 
         kw = dict()
         if self._has_rv:
-            kw['radial_velocity'] = self.radial_velocity
-
+            # Fill nans with zeros
+            fill = ~np.isfinite(self.radial_velocity)
+            rv = self.radial_velocity*1
+            rv[fill] = 0*u.km/u.s
```

Comment:
I think I'd like to go the same route with `.radial_velocity` as with distance: add a `get_radial_velocity()` method with a `fill_value` argument.

Reply:
`get_skycoord()` could then be called like:
```python
g.get_skycoord(distance=g.get_distance(fill_value=...),
               radial_velocity=g.get_radial_velocity(fill_value=...))
```
when there are nan or bad values in there.